.github/ISSUE_TEMPLATE/crash.yml

```
@@ -0,0 +1,73 @@
+name: Crash Report
+description: File a report for a crash in SuperTux.
+title: "[Crash]: "
+labels: ["type:crash"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this crash report!
+  - type: input
+    id: supertux-version
+    attributes:
+      label: SuperTux Version
+      description: The version of SuperTux, shown at the bottom-left corner on the title screen.
+      placeholder: ex. v0.6.3-1236-g7451121dd
+    validations:
+      required: true
+  - type: input
+    id: system-info
+    attributes:
+      label: System Information
+      description: Details about your operating system (like Linux distribution or Windows version) and CPU architecture (ex. 64-bit or 32-bit).
+      placeholder: ex. Windows 11 64-bit
+    validations:
+      required: true
+  - type: textarea
+    id: steps-to-reproduce
+    attributes:
+      label: Steps To Reproduce Crash
+      description: A clear and concise description of how the crash was achieved, since starting up the game.
+      placeholder: |
+        Describe the steps, leading to the crash, preferably with a numerated list.
+
+        1. ...
+        2. ...
+        3. ...
+    validations:
+      required: true
+  - type: textarea
+    id: debug-stacktrace
+    attributes:
+      label: Debugging Information (Stacktrace)
+      description: Include a stacktrace, leading to the crash. This can be obtained via a debugging utility.
+      placeholder: Paste the full stacktrace here.
+      render: shell
+    validations:
+      required: true
```

This shouldn't be required because you can't always get the debugging information if the program freezes for example
```suggestion
      required: false
```